parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -53,11 +53,10 @@ fi
 
 VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
 if [[ "${IS_VHD}" = true ]]; then
-    # TODO (09/13): uncomment when new VHD is published
-    # if [ -f $VHD_LOGS_FILEPATH ]; then
-    #     echo "Using VHD distro but file $VHD_LOGS_FILEPATH not found"
-    #     exit $ERR_VHD_FILE_NOT_FOUND
-    # fi
+    if [ ! -f $VHD_LOGS_FILEPATH ]; then
```

Comment:
@CecileRobertMichon note we need to change this test to "not file exists" (my bad, should have detected that in the original PR review of #1944